dev-aws/kafka-shared-msk/pubsub/mm2-big-remote-load-test.tf

```diff
@@ -51,8 +51,8 @@ resource "kafka_quota" "quota" {
   entity_name = "User:CN=pubsub/mirror-maker-load-test"
   entity_type = "user"
   config = {
-    # produce with 500 KB/broker
-    "producer_byte_rate" = "512000"
+    # produce with 5 KB/broker
+    "producer_byte_rate" = "512"
```

Comment:
out of curiosity, what was the problem with the 500kb/broker config?

Reply:
It wasn't enforced :). But we found out the problem eventually... the user attribute should not contain the "User:" prefix. Need to fix this in the tls-app module